FrontEnd/src/components/basics/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FunctionComponent, useState } from 'react';
+import { Form, Col, FormControlProps } from 'react-bootstrap';
+import styles from './TextArea.module.scss';
+import { Body2 } from '@basics';
+import cn from 'classnames';
+
+interface TextAreaProps extends FormControlProps {
+  maxLength: number;
+  label: string;
+  controlId: string;
+  contentPrefilled: string;
```

Comment:
nit: consider adding a comment explaining what this does. it doesn't seem to clear just from the name

Reply:
changed to defaultContent. comment should not be the savior for bad naming